content/docs/use-cases/versioning-data-and-model-files/index.md

```diff
@@ -1,28 +1,38 @@
 # Versioning Data and Model Files
 
-DVC enables versioning large files and directories such as datasets, data
-science features, and machine learning models using Git, but without storing the
-contents in Git.
+[Version control](https://en.wikipedia.org/wiki/Version_control) has become a
+staple in software engineering because it allows effective collaboration on
+source code. This means having a change history to go back to (commits),
+developing features in parallel (branching), assisted merging, peer-reviews
+(pull requests), tagging key revisions, etc. Imagine if we could use these
+features for data modeling!
```

Comment:
Is "data modeling" the right term? I'm trying to keep this use case generic, beyond ML or even data science.

Reply:
I guess that term is more related to databases (e.g. https://www.datasciencegraduateprograms.com/data-modeling/). I'll use "data science" for now. Cc @jeremydesroches have you noticed this term or related in searches? May have potential

Reply:
dvc.org had some impressions for ["data modeling" and related terms](https://search.google.com/search-console/performance/search-analytics?resource_id=sc-domain%3Advc.org&num_of_months=12&query=*data%20modeling&metrics=CLICKS%2CIMPRESSIONS). That's a start. Looking at overall search volume, "data modelling" and related terms have decent volume but the subject matter tends to the data architect role — not so much for data science or machine learning, etc.

Reply:
Thanks for checking. I went for something else in the end actually, after asking Elle about it. And now I found out how to use the Search Console haha so won't be bothering you with these basic questions much 😬